src/tests/test_tls_rfc8448.cpp

```diff
@@ -987,6 +987,7 @@ class Test_TLS_RFC8448_Client : public Test_TLS_RFC8448
                   "tls_verify_cert_chain",
                   "tls_verify_message"
                   });
+               result.require("certificate exists", ctx->certs_verified().size() > 0);
```

Comment:
This change is somewhat unrelated to the PR but at one point I broke the extension parsing which caused the list to become empty here, which resulted in the test segfaulting instead of failing nicely.